torch/csrc/jit/interpreter.cpp

```diff
@@ -726,8 +726,13 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
       if (num_outputs == 1) {
         future->markCompleted(stack.back());
       } else {
-        future->markCompleted(
-            Tuple::create(jit::last(stack, num_outputs).vec()));
+        ArrayRef<IValue> outputs = jit::last(stack, num_outputs);
```

Comment:
This replaced 6 characters with several lines. It obscures what is actually happening in this line. Not sure why something that is using the list has to call a `::impl` function either.